src/server/routes/configuration.js

```diff
@@ -0,0 +1,17 @@
+/* eslint camelcase: 0 */
```

Comment:
was this route already removed in staging? why is it being re-added?

Reply:
Oh good catch, this happened accidentally while resolving merge conflicts. Will remove.